core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreNewNavigationActivity.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.core.main;
+
+import android.content.Intent;
+import android.os.Bundle;
+import androidx.fragment.app.Fragment;
+import org.jetbrains.annotations.Nullable;
+import org.kiwix.kiwixmobile.core.R;
+import org.kiwix.kiwixmobile.core.base.BaseActivity;
+
+public abstract class CoreNewNavigationActivity extends BaseActivity {
+
+  @Override protected void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    super.setFragmentActivity(true);
```

Comment:
I switched to your branch, deleted this property in the base class and everything worked fine

Reply:
Interesting! Thanks for finding that out.

Reply:
I tried to make CustomMainActivity use the fragment and now I get the same crash ([butterknife can't find 'activity_main_root' with ID .... ](https://gist.github.com/Frans-Lukas/a0d2323fd2df15683d1442e62b1d7ea2)) unless I keep that property.

Reply:
Do a clean and it'll work.

How Butterknife works is in a class you say

```
@BindView(R.id.whatever)
MyView view
```
Then at build time it creates a class to call `findViewById` and cast it to the appropriate view and assign - or _inject_ - it into this field, hence the package visibility on this field

These generated classes can get out of sync though. You can check through it logically, it should only be trying to find a view for a class if you have declared it with `@BindView` in that class, seeing as `R.id.activity_main_root` is not bound in CustomMainActivity or any of its super classes then you can assume butterknife is just being a nob

Reply:
Oh, right! I am not used to having to clean build. Thank you.